### PR TITLE
Darker backgrounds

### LINKED ISSUE #4154: Adjust headline and standfirst background colour
Should be news[300] and news[200] rather than news[400] and news[300]

Screenshot 2022-03-07 at 11 39 18

### PR DESCRIPTION
**Pull request by oliverlloyd**
Fixes #4154

## What does this change?
This PR darkens the background used for a liveblog headline and standfirst

## Why?
To match the designs

### Before
Screenshot 2022-03-07 at 11 57 06

### After
Screenshot 2022-03-07 at 11 56 31

### CODE REVIEW

File: dotcom-rendering/src/web/lib/decidePalette.ts [hunk 1]
```diff
@@ -590,7 +590,23 @@ const backgroundHeader = (format: ArticleFormat): string => {
...
+				case ArticlePillar.Sport:
+					return sport[200];
+				case ArticlePillar.Lifestyle:
+					return lifestyle[200];
+				case ArticlePillar.Opinion:
+					return opinion[200];
+				case ArticleSpecial.Labs:
+					return labs[200];
+				case ArticleSpecial.SpecialReport:
+					return specialReport[200];
```
Comment (JamieB-gu) [inline dotcom-rendering/src/web/lib/decidePalette.ts:607]: Did we decide that we don't support these two as separate themes for liveblogs and that we'd fall back to news? Here's what `editorialPalette` does: https://github.com/guardian/dotcom-rendering/blob/72e90fe8963b4e8d8c87ad30678c61b536de288b/common-rendering/src/editorialPalette/background.ts#L82-L96

File: dotcom-rendering/src/web/lib/decidePalette.ts [hunk 2]
```diff
@@ -590,7 +590,23 @@ const backgroundHeader = (format: ArticleFormat): string => {
...
 	switch (format.design) {
 		case ArticleDesign.LiveBlog:
-			return pillarPalette[format.theme][300];
+			switch (format.theme) {
+				case ArticlePillar.News:
+					return news[200];
+				case ArticlePillar.Culture:
+					return culture[200];
+				case ArticlePillar.Sport:
+					return sport[200];
```
Comment (JamieB-gu) [inline dotcom-rendering/src/web/lib/decidePalette.ts:599]: Think this is `sport[100]`? Here's `editorialPalette` for this: https://github.com/guardian/dotcom-rendering/blob/72e90fe8963b4e8d8c87ad30678c61b536de288b/common-rendering/src/editorialPalette/background.ts#L82-L87

**Review by JamieB-gu — COMMENTED** [review]
Nice! 🎉

Couple minor points on specific colours.

File: dotcom-rendering/src/web/lib/decidePalette.ts (hunk 1 above)
Comment (oliverlloyd) [reply dotcom-rendering/src/web/lib/decidePalette.ts:607]: That sounds like a question for @HarryFischer !

File: dotcom-rendering/src/web/lib/decidePalette.ts (hunk 2 above)
Comment (oliverlloyd) [reply dotcom-rendering/src/web/lib/decidePalette.ts:599]: Ditto @HarryFischer

Comment (oliverlloyd) [reply dotcom-rendering/src/web/lib/decidePalette.ts:599]: Actually, I don't need to check with Harry, I can see in the designs that you're right - great spot!

File: dotcom-rendering/src/web/lib/decidePalette.ts (hunk 1 above)
Comment (HarryFischer) [reply dotcom-rendering/src/web/lib/decidePalette.ts:607]: @oliverlloyd I think we decided a while back to route Special Report blogs to use News palette for now. I can pull together some designs for a Special Report version in the future

**Comment by HarryFischer** [comment]
Colours changes look good to me @oliverlloyd

**Review by JamieB-gu — APPROVED** [review]

**Pull request merged**